map_name: "closed_road_shape.json"